### ISSUE TITLE
With Arduino mega 2560 will stuck in wifi.init()

### ISSUE THREAD

**Issue by e-lin** [sparkfun/SFE_CC3000_Library#16]
Hi,

I got an Arduino mega 2560 from [arduino.org](http://www.arduino.org/products/boards/4-arduino-boards/arduino-mega-2560), and I also have another Arduino UNO R3 board from arduino.cc

Following the Getting Started guide from README.md, I can successfully connect the Arduino UNO to Internet with TI CC3000 wifi shield and SFE_CC3000_Library. However, when changing to the Arduino Mega 2560 board, the process will stuck in wifi.init() and never come out.

With the WebClient example provided from SFE_CC3000_Library, the message from usb monitor is be like this:

```
---------------------------
SparkFun CC3000 - WebClient
---------------------------
```

And I never get

```
Something went wrong during CC3000 init!
```

Is there something else I need to do for Arduino mega 2560?

**Comment by e-lin** [comment]
It seems the same problem with [issue#4](https://github.com/sparkfun/SFE_CC3000_Library/issues/4) as it is also stuck at `wlan_start(0);` in SFE_CC3000.cpp which is invoked by `wifi.init()`
But I've tried to use [Adafruit_CC3000_Library](https://github.com/adafruit/Adafruit_CC3000_Library) as the latest post said, and have tried to connect the TI CC3000 wifi shield with jump wires as breakout. Both of those tries are still not working.

Anything I have to do with Arduino mega 2560?

**Comment by ShawnHymel** [comment]
You said the Adafruit library isn't working either? How are connecting the Mega to the CC3000? Are you using the ISP or SPI (pins 50-53)?

**Comment by e-lin** [comment]
No, the Adafruit library isn't working either. I did not use ISP or SPI. I followed this [hook up guide](https://learn.sparkfun.com/tutorials/cc3000-hookup-guide?_ga=1.226932568.413860516.1431500632) and only soldered headers on the both side of CC3000( SCL, SDA,..., TX, RX  and IOREF, RST,..., A4, A5). as picture showed below:
![test](https://cloud.githubusercontent.com/assets/12026254/10186349/41a55bac-678b-11e5-8f51-e3d9ade1120d.jpg)
